valhalla/userrequests/views.py

```diff
@@ -14,3 +19,50 @@ def get_queryset(self):
             return UserRequest.objects.filter(proposal__in=self.request.user.proposal_set.all())
         else:
             return UserRequest.objects.none()
+
+def parse_date(date_string):
```

Comment:
can't you just be happy I'm not using this? http://git.lco.gtn/projects/UT/repos/mnorbury_dateutil/browse